js/platform-window.js

```diff
@@ -159,12 +164,15 @@ class LeftMenu extends HTMLElement {
     async cloneWindow() {
         const bounds = await fin.me.getBounds();
         const layout = await fin.Platform.Layout.getCurrentSync().getConfig();
+        const customContext = await fin.Platform.getCurrentSync().getWindowContext();
         const snapshot = {
             windows: [
                 {
                     defaultWidth: bounds.width,
                     defaultHeight: bounds.height,
-                    layout
+                    layout,
+                    //getWindowContext actually returns the customContext option.
```

Comment:
Do you think it's worth renaming for consistency?

Reply:
what do you mean  ?

Reply:
Should we rename `options.customContext` to `options.context`?

`customContext` vs `windowContext` vs `context` vs `hostContext`
^ All these labels are used at some point. I think it's not terrible, especially since stuff like the theming use case makes me think that we may end up adding a notion of top level "platformContext" or something similar some day.

But maybe it's worth unifying names in the interest of keeping people from typing the wrong one.